config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryanalyticshubdataexchanges.bigqueryanalyticshub.cnrm.cloud.google.com.yaml

```diff
@@ -8,18 +8,15 @@ metadata:
     cnrm.cloud.google.com/managed-by-kcc: "true"
     cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
```

Comment:
Ah, I guess we can't NOTYET it because it was defined in Terraform .... it is alpha, but if we choose to remove it, let's do that in a different PR